packages/grafana-ui/src/components/Table/HeaderRow.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { HeaderGroup, Column } from 'react-table';
+import { DataFrame, Field } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { getTableStyles, TableStyles } from './styles';
+import { useStyles2 } from '../../themes';
+import { Filter } from './Filter';
+import { Icon } from '../Icon/Icon';
+
+export interface HeaderRowProps {
+  headerGroups: HeaderGroup[];
+  data: DataFrame;
+}
+
+export const HeaderRow = (props: HeaderRowProps) => {
+  const { headerGroups, data } = props;
+  const e2eSelectorsTable = selectors.components.Panels.Visualization.Table;
+  const tableStyles = useStyles2(getTableStyles);
+
+  return (
+    <div role="rowgroup">
+      {headerGroups.map((headerGroup: HeaderGroup) => {
+        const { key, ...headerGroupProps } = headerGroup.getHeaderGroupProps();
+        return (
+          <div
```

Comment:
Instead of `div` should this be a `thead`, in regards to a11y?

Reply:
Probably, though I didn't change this code I just moved it to it's own component.